ci/test/mkpipeline.py

```diff
@@ -110,6 +111,13 @@ def trim_pipeline(pipeline: Any) -> None:
                             find_compose_images(images, plugin_config["config"])
                         )
                         step.manual_inputs.add(plugin_config["config"])
+                    elif name == "./ci/plugins/mzconduct":
+                        comp = Composition.find(plugin_config["test"]).path
+                        config = comp / "mzcompose.yml"
+                        step.image_dependencies.update(
+                            find_compose_images(images, config)
+                        )
+                        step.manual_inputs.add(str(config))
```

Comment:
Totally optional (I'll get around to cleaning this up eventually if not) but we can ditch the hacky "#image" input format now (L94 above)! Also `manual_inputs` is shaping up to be a bad name, whoops. Maybe "extra_inputs" or something?